code/lib/cli/src/automigrate/index.ts

```diff
@@ -130,19 +179,39 @@ function getMigrationSummary(
   const failedFixesMessage =
     Object.keys(fixSummary.failed).length > 0
       ? `
-    ${chalk.bold('Migrations that failed:')}\n ${Object.entries(fixSummary.failed).reduce(
+    ${chalk.bold('Failed migrations:')}\n ${Object.entries(fixSummary.failed).reduce(
           (acc, [id, error]) => {
             return `${acc}\n${chalk.redBright(id)}:\n${error}\n`;
           },
           ''
         )}
-    \n`
+    `
+      : '';
+
+  const manualFixesMessage =
+    fixSummary.manual.length > 0
+      ? `
+      ${chalk.bold('Manual migrations:')}\n\n ${fixSummary.manual
+          .map((m) =>
+            fixResults[m] === FixStatus.MANUAL_SUCCEEDED ? chalk.green(m) : chalk.blue(m)
+          )
+          .join(', ')}
+    `
+      : '';
+
+  const skippedFixesMessage =
+    fixSummary.skipped.length > 0
+      ? `
+      ${chalk.bold('Skipped migrations:')}\n\n ${fixSummary.skipped
+          .map((m) => chalk.cyan(m))
+          .join(', ')}
+    `
       : '';
```

Comment:
the formatting on this is a bit of a nightmare, and likely only works by trial an error.

I do agree with @tmeasday that adding stories for these over time (that accurately demonstrate what is shown in the CLI ofc.) is a must-have for the future-us. Because I suspect changing something in here, will possibly change the whole formatting..

Reply:
I did experiment with making stories, but Storybook got all weird because `boxen` didn't work correctly there:
![2022-12-15 10 48 37](https://user-images.githubusercontent.com/1671563/207915113-23facf23-ea90-413c-a889-f528c5ac8727.gif)

I would love to do this properly in the future!